ETC.tsv

```diff
@@ -16521,17 +16521,17 @@ ETC_20151102_016522	Vedas Plateau
 ETC_20151102_016523	Koru Forest
 ETC_20151102_016524	Knidos Forest
 ETC_20151102_016525	Dadan Forest
-ETC_20151102_016526	Nevellet Mine 1F
-ETC_20151102_016527
+ETC_20151102_016526	Four Icon Castle
+ETC_20151102_016527	
 ETC_20151102_016528
-ETC_20151102_016529	Nevellet Mine 2F
+ETC_20151102_016529	Nevellet Mine 1F
 ETC_20151102_016530	Nevellet Mine 2F
 ETC_20151102_016531	Topes Mine 1F
 ETC_20151102_016532	Topes Mine 2F
 ETC_20151102_016533	Sicarius 1F
 ETC_20151102_016534	Sicarius 2F
-ETC_20151102_016535	Free Cleric Cus before the gas is released!
-ETC_20151102_016536	Cleric Cus
+ETC_20151102_016535	Free Cleric Kaus before the gas is released!
```

Comment:
Yeah I just thought Cus was an odd way to spell things. It's prolly a new NPC so no rush and ask IMC